examples/shader/gpu_readback.rs

```diff
@@ -145,7 +146,7 @@ fn prepare_bind_group(
     let bind_group = render_device.create_bind_group(
         None,
         &pipeline.layout,
-        &BindGroupEntries::single(buffers.gpu_buffer.as_entire_binding()),
+        &BindGroupEntries::single(buffers.gpu_buffer.buffer().unwrap().as_entire_binding()),
```

Comment:
As this is an example used for teaching it might be helpful to make this parameter it's own named variable and add a comment on why the unwrap won't fail.

Reply:
I was waiting on this PR to land so I can use the shiny new `binding()` function instead. https://github.com/bevyengine/bevy/pull/13667